src/main/java/app/Config.java

```diff
@@ -9,8 +8,6 @@
 import frameworks.gui.GUI;
 import frameworks.gui.SwingGUI;
 
-// UseCaseFactory
```

Comment:
pretty sure we need UseCaseFactory just per csc207 requirements

Reply:
agree with this!